src/scheduler/dbHelpers.ts

```diff
@@ -194,9 +194,11 @@ export async function updateIsAnnualReview(
       },
     };
     try {
-      logger.debug(`updating isAnnualReview for list item ${listItem.id}`);
-      await prisma.listItem.update(updateListItemPrismaStatement);
-      updatedListItems.push(listItem);
+      if (!listItem.isAnnualReview) {
+        logger.debug(`updating isAnnualReview for list item ${listItem.id}`);
+        await prisma.listItem.update(updateListItemPrismaStatement);
+        updatedListItems.push(listItem);
+      }
```

Comment:
Out of curiosity what issues where you seeing that lead to adding the if statement?

Reply:
The code isn't clear at this point but I think I found later the query does just search for !isAnnualReview. 

Code (at this point) looks like it possibly updates listItems even if is in isAnnualReview. 

More of a "just in case" or "to be clear" rather than fixing an issue.

Reply:
Shouldn't really update something if it's not being updated -- messes up updatedAt (which might affect debugging)